tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op.cc

```diff
@@ -304,6 +304,18 @@ class CuckooHashTableOfTensors final : public LookupInterface {
     return table_->export_values(ctx, value_dim);
   }
 
+  Status Save(OpKernelContext* ctx, const string filepath,
+              const size_t buffer_size) {
+    int64 value_dim = value_shape_.dim_size(0);
+    return table_->save(ctx, value_dim, filepath, buffer_size);
+  }
+
+  Status Load(OpKernelContext* ctx, const string filepath,
```

Comment:
LoadFromFile might be better for possible extending in the future.

Reply:
Accept